conda/environment.yml

```diff
@@ -10,13 +10,13 @@ dependencies:
   - cookiecutter
   - git
   - ipyleaflet
-  - jupyter-book=0.13
+  - jupyter-book>=0.13
```

Comment:
Just a note that the splash page is not using any components of JupyterBook. This long list of libraries is a relic of cloning the big website template and could use some scrubbing to make it tailored to the standalone splash page.

Reply:
> This long list of libraries is a relic of cloning the big website template and could use some scrubbing to make it tailored to the standalone splash page.

Agreed!